src/osd/OpRequest.cc

```diff
@@ -35,6 +35,7 @@ OpRequest::OpRequest(Message* req, OpTracker* tracker)
       request(req),
       hit_flag_points(0),
       latest_flag_point(0),
+      delayed_detail(nullptr),
```

Comment:
If the intent is to populate this for every event that uses mark_flag_point (that is, all but mark_sub_op_sent), let's name this last_event_detail.

Reply:
Yes